imagebuildah/build.go

```diff
@@ -181,7 +181,7 @@ func warnOnUnsetBuildArgs(node *parser.Node, args map[string]string) {
 				argHasValue = argFound[argName]
 			}
 			if _, ok := args[argName]; !argHasValue && !ok {
-				logrus.Warnf("missing %q build argument. Try adding %q to the command line", argName, fmt.Sprintf("--build-arg %s=<VALUE>", argName))
+				fmt.Fprintf(Err, "Warning: missing %q build argument. Try adding %q to the command line\n", argName, fmt.Sprintf("--build-arg %s=<VALUE>", argName))
```

Comment:
Something like (untested)
```
log := logrus.New()
log.SetOutput(Err)
log.SetLevel(logrus.GetLevel())
log.Warnf(...)
```
would probably be less disruptive.

Reply:
If we could do this one per connection, then this would solve our problem.

Reply:
We can try to wire in a way to pass a logger to some of our functions, but I don't really think we're going to be able to add a "send logrus output here" parameter to every function that logs things in the packages and libraries that we call into, and that's what'd be necessary to be sure we were catching everything.  Not that I know how our caller would filter and route things if all it could control was the output `io.Writer`.

Reply:
I say we do our best.  So that most of these messages get back to the caller.